examples/cisco/demo_vpc_plus.pp

```diff
@@ -16,7 +16,7 @@
 
 class ciscopuppet::cisco::demo_vpc_plus {
 
-  if platform_get() =~ /n(5|6|7)k/ {
+  if platform_get() =~ /n7k/ {
     if platform_get() == 'n7k' {
       $fabricpath_multicast_load_balance = true
       $port_channel_limit = false
```

Comment:
This one I left it because Deepak asked me, he says that once the support for 5k ans 6k is there, we just need to change this line rather than remove it altogether. we can discuss in today's meeting